release-notes/latest.md

```diff
@@ -69,3 +69,23 @@ Currently the only API action that can be limited in this way is `ListAllJobs` -
 #### <sub><sup><a name="5624" href="#5624">:link:</a></sup></sub> fix
 
 * Fixed a bug where fly would no longer tell you if the team you logged in with was invalid
+
+#### <sub><sup><a name="5192" href="#5192">:link:</a></sup></sub> fix
+
+* @evanchaoli changed the behaviour of the web to retry individual build steps that fail when a worker disappears.
+
+#### <sub><sup><a name="5549" href="#5549">:link:</a></sup></sub> fix
+
+* Added a new Wrappa that returns HTTP 409 for endpoints listed in concourse/rfc#33 when the requested pipeine is archived
+
+#### <sub><sup><a name="5576" href="#5576">:link:</a></sup></sub> fix
+
+* @dcsg added support for AWS SSM for `var_sources`
+
+#### <sub><sup><a name="5575" href="#5575">:link:</a></sup></sub> feature
+
+* Added tracing to the lidar component, where a single trace will be emitted for each run of the scanner and the consequential checking that happens from the checker. The traces will allow for more in depth monitoring of resource checking through describing how long each resource is taking to scan and check. #5575
```

Comment:
It looks like this is the only item that has the link (#5575) tacked on to the end. We should take that out, since the heading will link to the issue already.

It might be easier to understand if we split it into multiple bullet points each expressing one idea, and link to lidar for those unfamiliar with it, like this:

```
#### <sub><sup><a name="5575" href="#5575">:link:</a></sup></sub> feature
* Added tracing to the [lidar component](https://github.com/concourse/concourse/tree/master/atc/lidar)
* A single trace will be emitted for each run of the scanner and the consequential checking that happens from the checker. 
* By describing how long each resource is taking to scan and check, these traces allow for more in depth monitoring of resource checking.
```

Reply:
Looking at past releases it seems like we usually stick the issue number at the end, otherwise a link to the issued doesn't appear anywhere
https://github.com/concourse/concourse/releases

Reply:
I'm going to add the issue numbers to the end of each note.